src/tool_help.c

```diff
@@ -94,6 +94,8 @@ static const struct helptxt helptext[] = {
    "Convert LF to CRLF in upload"},
   {"    --crlfile <file>",
    "Get a CRL list in PEM format from the given file"},
+  {"    --curves <colon-separated-list of curves to select>",
+   "during SSL session setup"},
```

Comment:
this doesn't match the text in `curves.d` file, suggesting this was not generated as the comment above this list says...

Reply:
Got it. Thanks for the pointer. Will update.